app/components/ui/CalculatorInput.tsx

```diff
@@ -74,175 +74,276 @@ const CalculatorInput = () => {
+                          className="radio-label-style"
+                        >
+                          Medium (2%)
+                        </Label>
                       </div>
                       <div className="flex items-center space-x-2">
-                        <RadioGroupItem value="0.0375" id="radio-option-high" />
-                        <Label htmlFor="radio-option-high">High (3.75%)</Label>
+                        <RadioGroupItem
+                          value="0.0375"
+                          id="radio-option-high"
+                          className="radio-button-style"
+                        />
+                        <Label
+                          htmlFor="radio-option-high"
+                          className="radio-label-style"
+                        >
+                          High (3.75%)
+                        </Label>
                       </div>
                     </RadioGroup>
                   </FormControl>
-                  <FormDescription>
-                    Select a level for maintenance spend, as a percentage of
-                    house cost.
-                  </FormDescription>
+                  <FormDescription>What is this?</FormDescription>
                   <FormMessage />
                 </FormItem>
               )}
             />
 
-            <Button type="submit">Submit</Button>
+            <div className="grid grid-cols-2 gap-2">
+              <div className="flex flex-col">
+                <span className="h3-style inaccessible-input-style">
+                  Mortgage interest rate
+                </span>
+                <span className="inaccessible-input-style">6%</span>
+              </div>
+              <div className="flex flex-col">
+                <span className="h3-style inaccessible-input-style">
+                  Mortgage term
+                </span>
+                <span className="inaccessible-input-style">25 years</span>
+              </div>
+              <div className="flex flex-col">
+                <span className="h3-style inaccessible-input-style">
+                  Mortgage deposit
+                </span>
+                <span className="inaccessible-input-style">25 years</span>
+              </div>
+            </div>
+
+            <Button type="submit" className="calculate-button-style">
+              Calculate
+            </Button>
           </form>
         </Form>
       </div>
     );
   }
-  
+
   if (view === "loading") {
     return (
       <div className="flex items-center justify-center h-screen text-black mt-5">
         <ClipLoader color="black" size={50} />
       </div>
     );
-  }
-  
-  if (view === "dashboard" && data) {
-    return <Dashboard data={data} />;
+  } else if (view === "dashboard" && data) {
+    const formValues = form.getValues();
+    return (
+      <Dashboard
+        processedData={data}
+        inputData={formSchema.parse(formValues)}
+      />
+    );
```

Comment:
```suggestion
  }
  
  if (view === "dashboard" && data) {
    const formValues = form.getValues();
    return (
      <Dashboard
        processedData={data}
        inputData={formValues}
      />
    );
```

At this point, the values are already parsed so we don't need to repeat this 👍

Reply:
Hopefully fixed thanks @DafyddLlyr